app/furniture/marketplace/marketplaces/show.html.erb

```diff
@@ -1,2 +1,2 @@
 <%- breadcrumb :marketplace, marketplace %>
-<%= render Marketplace::Component.new(marketplace: marketplace) %>
+<%= render Marketplace::MarketplaceComponent.new(marketplace: marketplace) %>
```

Comment:
This is broken in main; where if you go to a marketplace directly there will be an exception. No one is doing that right now so it's not that big a deal but 😭.

I'll split this to it's own PR in a bit.

Reply:
https://github.com/zinc-collective/convene/pull/1492